src/guides/aaa_general/night-action-order.mdx

```diff
@@ -0,0 +1,34 @@
+---
+name: Basics
+menu: Night Action Order
+route: /night-action-order
+---
+
+# Order of Night Actions
+
+The different abilities roles have at night may influence each other. How they do is determined by the order in which the night actions are defined to take place. The order of night actions is:
+
+1. Redirects (Succubus)
+1. Roleblocks (Direwolf, Courtesan, Manacles of Spite)
+1. Protection visits (Protector, Huntsman, Shaman, Armour, Heavy Shield)
+1. Most visits (Too many to list: all visits not belonging in another category, including items)
+1. Item thefts from abilities and potions
+1. Kills (Killing werewolf (Werewolf/Alphawolf/Shapeshifter), Vampire, Witch, Militia), including resulting item thefts
+1. Report visits to Stalkers, Harlots, Familiar
+1. Passing of items
+1. Swap Identities (Djinn, Shapeshifter)
+1. Report kills to village
+1. Report revives to village
+
+Multiple actions that fall in the same category are executed more or less simultaneously. Conflicts between such actions are resolved in the following manner:
+
+- Redirects and identity swaps occur in a determinsitic, server-defined order, which may cause them to be unpredictable if two or more such actions
+  target the same player.
+- If a player is roleblocked multiple times, they will receive one message for each roleblock, but receive no additional effect.
+- Protection visits stack - the player is protected by all sources protecting them, and if a player is attacked, all associated effects trigger.
```

Comment:
Unsure if "stack" is the best word to use here

Reply:
Agree, it makes it sound like you're protected multiple times.

Reply:
Good point - I will adjust that. I think I'll use the word "overlap", as the player can't get any additional benefit from being protected multiple times.

Reply:
Corrected to "overlap"